README.md

```diff
@@ -1,15 +1,16 @@
 # GUDocs 2
 
-Google Docs to S3 uploader
+The gudocs tool allows visuals and interactive journalist to easily create JSON datasources for interactives from Google Docs and Sheets.
 
-* Google Docs -> ArchieML -> S3
-* Google Sheets -> CSV -> JSON -> S3
+For example, the [Men's football transfer window](https://www.theguardian.com/football/ng-interactive/2023/feb/01/mens-transfer-window-january-2023-premier-league-bundesliga-ligue-1-la-liga-serie-a) article is powered from [a JSON file](https://interactive.guim.co.uk/2023/tw-men-winter-2023/transfersData.json). This JSON file is generated from a spreadsheet by the gudocs tool.
 
-Description of how users users interact with the service
+Journalists interact with the tool through the web interface at https://gudocs.gutools.co.uk. From this interface they can publish a document from Google Docs to S3. All documents that are shared with the relevant google service account are published automatically to the `docsdata-test` folder when they are modified.
```

Comment:
I don't know if the name of the Google Service Account should be treated as confidential - maybe so? If it's not confidential, we could put it in brackets after "the relevant google service account" - alternatively, if it _is_ confidential, maybe link to a private document that contains that detail.

Should there be a bit more explanation of the use case of the two modes of operation described here? Am I right in thinking the intention is that public-facing changes are only ever pushed to the public when explicitly published using the UI at https://gudocs.gutools.co.uk - while automatic-publication to `docsdata-test` is a form of _preview_, where staff can check what their latest changes look like before explicitly publishing them?

Reply:
The service account is listed in the UI. It would be inconvenient for it to be shared, since it could be a source of spam. I'll update the README to explain where to see the service account.

The docsdata-test is publicly available too, and indeed some interactives are setup to read from docsdata-test since it live updates.